integrations/gitlab/.port/spec.yaml

```diff
@@ -42,5 +42,15 @@ configurations:
         \"subgroup_events\",
         \"confidential_issues_events\"]"
     sensitive: true
+saas:
+  enabled: true
+  oauthConfiguration:
+    requiredSecrets:
+      - name: tokenMapping
+        value: '{(.oauthData.accessToken): ["**"]} | @json'
+        description: '"Token mapping for Gitlab OAuth2 integration"'
+    valuesOverride:
+      appSpec:
+        minimumScheduledResyncInterval: '2h'
```

Comment:
is `minimumScheduledResyncInterval` still needed? I thought we decided to handle it in port